### PR TITLE
Fix pointer events on FF

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: common/src/main/webapp/sass/visualization.scss [hunk 1]
```diff
@@ -138,12 +138,14 @@ $brightness: var(--aladin-image-brightness);
 
   .circle-target {
     display: none;
+    pointer-events: fill;
```
Comment (hugo-vrijswijk) [inline common/src/main/webapp/sass/visualization.scss:141]: What is the difference in this case between the default (`auto`/`visiblePainted`) and `fill`?

File: common/src/main/webapp/sass/tooltips.scss [hunk 2]
```diff
@@ -42,7 +42,7 @@
 }
 
 .svg-target-tooltip-area {
-  pointer-events: bounding-box;
```
Comment (hugo-vrijswijk) [inline common/src/main/webapp/sass/tooltips.scss:45]: I'm wondering if there's some sort of s?css linting we can set up that warns for non-standard usage

Comment (cquiroz) [reply common/src/main/webapp/sass/tooltips.scss:45]: Yeah that would be useful

File: common/src/main/webapp/sass/visualization.scss (hunk 1 above)
Comment (cquiroz) [reply common/src/main/webapp/sass/visualization.scss:141]: I tried several combinations that would work on FF and Chrome and `auto/visiblePainted` didn't. Couldn't find any documentation that would indicate why they are different

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**